docs/website/docs/developers/debugging/gpu.md

```diff
@@ -0,0 +1,177 @@
+---
+hide:
+  - tags
+tags:
+  - GPU
+  - CUDA
+  - Metal
+  - ROCm
+  - Vulkan
+icon: material/chip
```

Comment:
nit: maybe use `icon: octicons/bug-16` here to match the surrounding pages instead, to reduce the visual noise. That, or reorder the titles (preferring to keep them alphabetical, so by renaming one of the pages...)
```suggestion
icon: octicons/bug-16
```

![image](https://github.com/openxla/iree/assets/4010439/e28d7f27-e1a9-44a1-84a1-11a00affdeed)